frontend/pages/queries/edit/EditQueryPage/EditQueryPage.tsx

```diff
@@ -101,6 +103,16 @@ const EditQueryPage = ({
     false
   );
 
+  const {
+    data: appConfig, // TODO - pass to savequery modal
```

Comment:
It looks like you passed them through, can we remove these comments?

Reply:
ah, yes! good catches